api-channel-issue-tracker/src/main/java/com/synopsys/integration/alert/api/channel/issue/model/IssueVulnerabilityDetails.java

```diff
@@ -45,4 +52,18 @@ public boolean areAllComponentVulnerabilitiesRemediated() {
         return vulnerabilitiesDeleted;
     }
 
+    public Optional<String> getHighestVulnerabilityAddedOrUpdated() {
+        return getHighestVulnerability(Stream.of(vulnerabilitiesAdded, vulnerabilitiesUpdated)
+                                           .flatMap(Collection::stream)
+                                           .collect(Collectors.toList()));
```

Comment:
Can we store the `List` in a variable? It's a little confusing nesting the whole `Stream` in the method call.